tests/gtests/test_SW_Carbon.cc

```diff
@@ -40,34 +40,29 @@
 
 
 namespace {
-  TimeInt simendyr = SW_All.Model.endyr + SW_All.Model.addtl_yr;
-
   // Test the SW_Carbon constructor 'SW_CBN_construct'
-  TEST(CarbonTest, Constructor) {
+  TEST_F(AllTest, CarbonConstructor) {
     int x;
 
     SW_CBN_construct(&SW_All.Carbon);
 
     // Test type (and existence)
     EXPECT_EQ(typeid(x), typeid(SW_All.Carbon.use_wue_mult));
     EXPECT_EQ(typeid(x), typeid(SW_All.Carbon.use_bio_mult));
-
-    // Reset to previous global state
-    Reset_SOILWAT2_after_UnitTest();
   }
 
 
   // Test reading yearly CO2 data from disk file
-  TEST(CarbonTest, ReadInputFile) {
-    TimeInt year;
+  TEST_F(AllTest, ReadCarbonInputFile) {

```

Comment:
Nice change from test name "ReadInputFile" to "ReadCarbonInputFile"! Since every test is now based on "AllTest" fixture -- we loose the name of the test suit. Please add such information as done here throughout